### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by pavinthan**
Dev dependencies versions updated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,10 +12,10 @@
     "devDependencies": {
         "axios": "^0.17",
         "bootstrap-sass": "^3.3.7",
-        "cross-env": "^5.1",
-        "jquery": "^3.2",
-        "laravel-mix": "^1.0",
+        "cross-env": "^5.1.1",
```
Comment (GrahamCampbell) [inline package.json]: This change doesn't do anything?

File: package.json [hunk 2]
```diff
@@ -12,10 +12,10 @@
...
+        "jquery": "^3.2.1",
```
Comment (GrahamCampbell) [inline package.json]: This change doesn't do anything?

**Comment by pavinthan** [comment]
Thank you guys.

**Comment by pavinthan** [comment]
@taylorotwell it's my small suggestion may you please give some short description when you close any PR because it's may help us to improve the way to create PR in future.

**Comment by GrahamCampbell** [comment]
@pavinthan Please see my comments. The explain why this wasn't merged.

**Comment by GrahamCampbell** [comment]
This pull request doesn't do anything. Those version constraints already resolve to the newest versions.

**Comment by pavinthan** [comment]
Thanks @GrahamCampbell and @taylorotwell, I understood.

**Pull request closed without merging**